blog/_posts/2017-12-11-Thanks-for-diversifying-tech-with-us.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Thanks for diversifying tech with us!
+layout: post
+created_at: Mon December 11 2017
```

Comment:
The date should be changed before we merge it — on Monday we will publish the “project submissions are open!” blog post, so we could publish this on Tuesday? What do you think @inescoelho?

Reply:
How about Wednesday? The project submission opening is a big thing, it might be better to give it a day or two before publishing/tweeting something else!

Reply:
@inescoelho sounds great to me! 👍